misc/userscripts/qute-pass

```diff
@@ -211,11 +226,41 @@ def extract_username(target, pattern):
         raise CouldNotMatchUsername("Pattern did not contain capture group, please use capture group. Example: (.*)")
 
 
+def extract_secret(selection):
+    secret = None
+    # If username-target is path and user asked for username-only, we don't need to run pass.
+    # Or if using otp-only, it will run pass on its own.
+    if not (arguments.username_target == 'path' and arguments.username_only) and not arguments.otp_only:
+        secret = pass_(selection)
+    username_target = selection if arguments.username_target == 'path' else secret
+
+    if arguments.username_only:
+        fake_key_raw(extract_username(username_target, arguments.username_pattern))
+    elif arguments.password_only:
+        fake_key_raw(extract_password(secret, arguments.password_pattern))
+    elif arguments.otp_only:
+        fake_key_raw(pass_otp(selection))
+    else:
+        # Enter username and password using fake-key and <Tab> (which seems to work almost universally), then switch
+        # back into insert-mode, so the form can be directly submitted by hitting enter afterwards
+        fake_key_raw(extract_username(username_target, arguments.username_pattern))
+        qute_command('fake-key <Tab>')
+        fake_key_raw(extract_password(secret, arguments.password_pattern))
+
+    if arguments.insert_mode:
+        qute_command('mode-enter insert')
+
+
 def main(arguments):
     if not arguments.url:
         argument_parser.print_help()
         return ExitCodes.FAILURE
 
+    if arguments.password_store is None:
+        # `pass` and `gopass` use different default password store directories
+        default = '~/.local/share/gopass/stores/root' if arguments.mode == "gopass" else '~/.password_store'
```

Comment:
```suggestion
        default = '~/.local/share/gopass/stores/root' if arguments.mode == "gopass" else '~/.password-store'
```
I think this underscore here is the culprit

Reply:
Yeah, I'm fixing this one now.